stock_picking_operation_quick_change/tests/test_stock_picking_operation_quick_change.py

```diff
@@ -72,26 +74,18 @@ def setUp(self):
         )
 
     def qty_on_hand(self, product):
-        wiz = self.env["stock.inventory"].create(
+        wiz = self.env["stock.quant"].create(
             {
-                "name": "Stock Inventory",
-                "product_ids": [(4, product.id, 0)],
-                "line_ids": [
-                    (
-                        0,
-                        0,
-                        {
-                            "product_id": product.id,
-                            "product_uom_id": product.uom_id.id,
-                            "product_qty": 200,
-                            "location_id": self.warehouse.lot_stock_id.id,
-                        },
-                    ),
-                ],
+                "name": "Packages",
```

Comment:
might want to look at https://github.com/odoo/odoo/commit/bdcb3d192be1e01e1141aa09c60027337009a67b